.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,8 @@
 blank_issues_enabled: false
 contact_links:
-    - name: Idea/Feature Request
+    - name: Idea Request
```

Comment:
"Idea Request" sounds weird. Should it just be "Idea"? Or maybe we should have "Small feature request" vs "Larger project idea"? I don't know how other projects do this.